### ISSUE TITLE
Very simplistic embedding projector example needed

### ISSUE THREAD

**Issue by luggie** [tensorflow/tensorboard#4276]
All available examples use default data sets and derive supervised learning models from it.
What I'd need is way simpler. I want to display data: already made embeddings, so nothing more but mere arbitrary vectors and according labels in TSV, numpy array or numpy array binary format (or literary anything else, pd.df's, handwritten notices, ancient runes)
```
from tensorboard.plugins import projector
LOG_DIR = '/some/path/where/embedding/is/stored'
config = projector.ProjectorConfig()
embedding = config.embeddings.add()
...
embedding.metadata_path = 'metadata.tsv'
projector.visualize_embeddings(LOG_DIR, config)
```

**Comment by rmothukuru** [comment]
@luggie,
A complete example for Embedding Projector is mentioned in the [Tensorboard Documentation](https://www.tensorflow.org/tensorboard/tensorboard_projector_plugin). 

Please let me know if this is what you are looking for. Thanks

**Comment by psybuzz** [comment]
In case you are still looking for a minimal example, the following worked for me:
```python
import os
import tensorflow as tf
from tensorboard.plugins import projector

# Assume a logdir exists
LOG_DIR = "./logs"

# 1) Save a .tsv with labels, line by line
METADATA_FNAME = "meta.tsv"
labels = ['excess', 'plant', 'understand']
with open(os.path.join(LOG_DIR, METADATA_FNAME), "w+") as f:
    for label in labels:
        f.write("{}\n".format(label))

# 2) Save the embedding in a checkpoint.
weights = tf.Variable([[1,2,3], [1,3,3], [3,2,3]], name="my weights var")
checkpoint = tf.train.Checkpoint(embedding=weights)
checkpoint.save(os.path.join(LOG_DIR, "embedding.ckpt"))

# 3) Configure the projector
config = projector.ProjectorConfig()
config.embeddings.add(
    tensor_name = "embedding/.ATTRIBUTES/VARIABLE_VALUE",
    metadata_path = METADATA_FNAME,
)
projector.visualize_embeddings(LOG_DIR, config)

```

I agree that the docs on the TF website are very non-minimal, and seem to suggest to Keras is required to use the Projector, when it is not.

Since the core issue of improving the Projector docs has been around in https://github.com/tensorflow/tensorboard/issues/86, I'm closing this issue as a duplicate.  If you would like to contribute or have ideas on the Projector docs, please comment on the other issue!
